File: JAVA/P001/src/Factorial.java
import java.util.Scanner;

public class Factorial {

	public static void main(String[] args) {
		int n;
		Scanner scanner = new Scanner(System.in);
		n = scanner.nextInt();
		System.out.print(n);
		Ex factorial = new Ex();
		System.out.println("의 누승은 " + factorial.calFact(n) + " 입니다.");
		}
}
class Ex{
	int Fact = 1;
	public int calFact(int n) {
		while(n > 0) {
			Fact = n * Fact;
			n = n - 1;
		}
		return Fact;
	}
}

File: C/P001/Factorial.c
#include <stdio.h>
void main() {
	int n, Fact = 1;

	scanf_s("%d", &n);
	printf("%d", n);
	Fact = factorial(n);
	printf("의 누승은 %d이다\n", Fact);
	system("pause");
}
int factorial(int n) {
	int Fact = 1;

	while (n > 0) {
		Fact = n * Fact;
		n = n - 1;
	}
	return Fact;
}